form-designer/form_designer/form_designer.py

```diff
@@ -92,3 +92,6 @@
     title=responses_title(),
     on_load=ResponsesState.load_responses,
 )
+
+# Create the database if it does not exist (hosting service does not migrate automatically)
+rx.Model.migrate()
```

Comment:
Does the hosted version use an external database? Should we just migrate it offline?

Reply:
the hosted version just uses a local sqlite file that will get blown away when the instance restarts.

i'm pretty sure the hosting service excludes `reflex.db` right?

Reply:
Yeah it excludes it - makes sense.